pydantic/validators.py

```diff
@@ -70,6 +70,12 @@ def str_validator(v: Any) -> Optional[str]:
         raise errors.StrError()
 
 
+def strict_str_validator(v: Any) -> str:
```

Comment:
I think this is not used, above you're using `make_arbitrary_type_validator(str)`

Reply:
done, you changed it while I was reviewing.